src/test/ui/lint/lint-non-uppercase-statics.stderr

```diff
@@ -14,7 +14,7 @@ error: static variable `bar` should have an upper case name
   --> $DIR/lint-non-uppercase-statics.rs:6:12
    |
 LL | static mut bar: isize = 1;
-   |            ^^^ help: convert the identifier to upper case: `BAR`
+   |            ^^^ help: convert the identifier to upper case (notice the capitalization): `BAR`
```

Comment:
The change in diagnostic seems counterproductive in this case -- it's is just repetition.

Reply:
I've modified the logic to only apply when the suggestion has an ASCII character that differs in capitalization and is in a whitelist of confusable chars.